### PR TITLE
chore: Tags Versioning

### PR DESCRIPTION
**Pull request by eedygreen**
Tags Versions

## Description
Added Tags Versions

Closes: #121 

## How Has This Been Tested? Testing details.
Tested. Tagged branch name

## Types of changes
- [ x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: .github/workflows/deploy_testnet.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ jobs:
         with:
           context: .
           push: true
-          tags: ${{ env.REGISTRY }}/${{ github.repository }}:${{ env.TAG }}
+          tags: ${{ env.REGISTRY }}/${{ github.repository }}:${{ env.TAG }}-${{ github.ref_name }}
```
Comment (mpetrun5) [inline .github/workflows/deploy_testnet.yml:48]: We keep this as is.

**Review by mpetrun5 — CHANGES_REQUESTED** [review]

File: .github/workflows/deploy_stage.yml [hunk 2]
```diff
@@ -45,7 +45,7 @@ jobs:
         with:
           context: .
           push: true
-          tags: ${{ env.REGISTRY }}/${{ github.repository }}:${{ env.TAG }}
+          tags: ${{ env.REGISTRY }}/${{ github.repository }}:${{ env.TAG }}-${{ github.ref_name }}
```
[the remark this answers is not in the document]
Comment (eedygreen) [reply .github/workflows/deploy_stage.yml]: added

File: .github/workflows/deploy_testnet.yml [hunk 3]
```diff
@@ -47,6 +47,13 @@ jobs:
           push: true
           tags: ${{ env.REGISTRY }}/${{ github.repository }}:${{ env.TAG }}
 
+      - name: tag version / push docker image into ghcr
+        id: build-and-push
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          push: true
+          tags: ${{ env.REGISTRY }}/${{ github.repository }}:${{ env.TAG }}-${{ github.ref_name }}
```
Comment (mpetrun5) [inline .github/workflows/deploy_testnet.yml:56]:
```suggestion
          tags: ${{ env.REGISTRY }}/${{ github.repository }}:${{ github.ref_name }}
```

**Review by mpetrun5 — APPROVED** [review]

File: .github/workflows/deploy_testnet.yml (hunk 3 above)
Comment (eedygreen) [reply .github/workflows/deploy_testnet.yml]: Thanks

**Pull request merged**